docs/javascript/v2/how-to-guides/extend-capabilities/plugins/krisp-noise-cancellation.mdx

```diff
@@ -14,6 +14,12 @@ This guide provides an overview of usage of the noise suppression plugin of 100m
 
 Minimum version requirement for `hms-video-store` - 0.11.7
 
+<Note type='warning'>
+**IMPORTANT** <br/>
+Enable Noise Cancellation in the template configuration. Learn more on enabling this feature from [here](/get-started/v2/get-started/features/noise-cancellation#enabling-the-noise-cancellation)
```

Comment:
Sentence update:

Learn more about enabling this feature [here](https://github.com/get-started/v2/get-started/features/noise-cancellation#enabling-the-noise-cancellation).